scripts.py

```diff
@@ -539,6 +539,8 @@ def refresh_metadata(self, identifier):
             provider = OverdriveBibliographicCoverageProvider
         elif identifier.type==Identifier.AXIS_360_ID:
             provider = Axis360BibliographicCoverageProvider
+        elif identifier.type==Identifier.ENKI_ID:
+            provider = EnkiBibliographicCoverageProvider
```

Comment:
You're not importing the right class here. At any rate, this script doesn't work very well so I'm not sure it's worth fixing. Maybe just take this whole thing out.